ChatApp/Janus/Janus.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  File.swift
+//  ChatApp
+//
+//  Created by Kalvin Dela Raga on 1/27/21.
+//
+
+import UIKit
+import Firebase
+
+class Janus {
+    
+    let window: UIWindow
+    
+    private var listener: AuthStateDidChangeListenerHandle?
+    
+    init(window: UIWindow) {
+        self.window = window
+    }
+    
+    func start() {
+        listener = Auth.auth().addStateDidChangeListener { (auth, user) in
```

Comment:
maybe it's possible to add [weak self] it might hold strong reference and this class wont be deallocated